nucypher/characters/lawful.py

```diff
@@ -602,7 +630,7 @@ def _try_orient(self, treasure_map, alice_verifying_key):
             raise  # TODO: Maybe do something here?  NRN
 
     def get_treasure_map(self, alice_verifying_key, label):
```

Comment:
Should be `relayer_verifying_key`?

Reply:
I agree, "publisher" provides more intuition to what the character does. Do we still use names for characters? Pablo the Publisher?

Reply:
Renamed `relayer_verifying_key` -> `publisher_verifying_key` for now.